src/wallet/rpcwallet.cpp

```diff
@@ -26,8 +26,7 @@
 #include "wallet/feebumper.h"
 #include "wallet/wallet.h"
 #include "wallet/walletdb.h"
-
-#include <init.h>  // For StartShutdown
+#include "init.h"  // For StartShutdown
```

Comment:
wallet/?

Reply:
No, `src` one, but it includes `src/wallet` instead.

Reply:
This change to a relative include is wrong. We want wallet in the include root, not init in `wallet/`